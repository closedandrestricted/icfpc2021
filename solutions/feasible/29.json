{"vertices":[[19,26],[34,2],[0,5],[19,26]]}