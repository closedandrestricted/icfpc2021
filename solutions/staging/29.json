{"vertices":[[30,14],[11,35],[11,35],[7,7]]}