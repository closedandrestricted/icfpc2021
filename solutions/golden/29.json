{"vertices":[[8,3],[12,31],[12,31],[32,11]]}